pkg/testutils/random/random.go

```diff
@@ -0,0 +1,25 @@
+package random
+
+import "math/rand"
+
+const (
+	randomNameLength        = 8
+	randomNameCharSet       = "abcdefghijklmnopqrstuvwxyz"
+	randomPortUpperBoundary = 65535
+)
+
+// Name creates a random string [a-z] of len 8.
+func Name() string {
+	b := make([]byte, randomNameLength)
+	for i := range b {
+		//nolint:gosec // random number generator sufficient for testing purposes
+		b[i] = randomNameCharSet[rand.Intn(len(randomNameCharSet))]
+	}
+	return string(b)
+}
+
+// Port creates a random int64 in range [1, 65535].
+func Port() int64 {
+	//nolint:gosec // random number generator sufficient for testing purposes
+	return int64(rand.Intn(randomPortUpperBoundary) + 1)
+}
```

Comment:
I have moved the existing `RandomName()` func to this helper for better re-use (commit https://github.com/kyma-project/lifecycle-manager/pull/1349/commits/afc3a0a46699e7f06ed62fd2b9b52aef17cd3da4).

It was part of `KymaBuilder` but already used elsewhere. Originally, I wanted to promote it to `testutils` package, but this wasn't simply possible because it would have introduced circular dependencies. Hence, I put it into a separate `testutils/random` package.